README.md

```diff
@@ -248,6 +248,8 @@ python3 setup.py install
     * [ ] `room count`
     * [ ] `room top-complexity`
     * [ ] `room top-members`
+    * [x] [`room block`](https://matrix-org.github.io/synapse/v1.73/admin_api/rooms.html#block-or-unblock-a-room)
+    * [x] [`room block-status`](https://matrix-org.github.io/synapse/v1.73/admin_api/rooms.html#get-block-status)
```

Comment:
i think the subcommand names are pretty self explanatory, so linking to the docs might not be necessary. as such, i'll remove it (in a separate commit so that it is revertible).